Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -333,8 +333,23 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
     connect(m_ui->m_assetBrowserTreeViewWidget, &AzAssetBrowser::AssetBrowserTreeView::ClearTypeFilter,
         m_ui->m_searchWidget, &AzAssetBrowser::SearchWidget::ClearTypeFilter);
 
-    connect(m_assetBrowserModel, &AzAssetBrowser::AssetBrowserModel::RequestOpenItemForEditing,
-        m_ui->m_assetBrowserTreeViewWidget, &AzAssetBrowser::AssetBrowserTreeView::OpenItemForEditing);
+    connect(
+        m_assetBrowserModel,
+        &AzAssetBrowser::AssetBrowserModel::RequestOpenItemForEditing,
+        this,
+        [this](const QModelIndex& index)
+        {
+            m_ui->m_assetBrowserTreeViewWidget->OpenItemForEditing(index);
+        });
+
+    connect(
+        m_assetBrowserModel,
+        &AzAssetBrowser::AssetBrowserModel::RequestThumbnailviewUpdate,
+        this,
+        [this]
+        {
+            m_ui->m_thumbnailView->UpdateThumbnailview();
+        });
```

Comment:
If you make `UpdateThumbnailview` a slot, then you don't have to create a lambda and can just connect it directly like:
```suggestion
    connect(m_assetBrowserModel, &AzAssetBrowser::AssetBrowserModel::RequestThumbnailviewUpdate,
         m_ui->m_thumbnailView, &AssetBrowser::AssetBrowserThumbnailView::UpdateThumbnailview);
```

Reply:
Sounds good, I've changed it to a slot